ios/Mattermost/MattermostManaged.m

```diff
@@ -28,6 +28,49 @@ - (void)stopObserving {
   hasListeners = NO;
 }
 
+- (BOOL)hasSafeAreaInsets {
```

Comment:
Why is this in MattermostManaged? I thought that file is all about EMM stuff, so it seems weird that we'd include this here.

Reply:
It seemed completely unnecessary creating yet another module just for this, open to suggestions if you want it elsewhere

Reply:
I agree that it's kind of unnecessary for this to have its own module, but the rest of the code in here is focused on EMM stuff so we shouldn't be adding unrelated stuff to this module if we're enforcing proper separation of concerns. That said, I'm fine treating this as a general purpose module for now since we don't have much native code in the app, even though the name is misleading.